examples/pipelines/finetune_stable_diffusion/components/image_embedding/fondant_component.yaml

```diff
@@ -0,0 +1,23 @@
+name: Image embedding
+description: Component that embeds images using CLIP
+image: ghcr.io/ml6team/image_embedding:latest
+
+input_subsets:
+  images:
+      fields:
+        data:
+          type: binary
+
+output_subsets:
+  embeddings:
+    fields:
+      data:
+        type: float32_list
+
+args:
+  model_id:
+    description: Model id on the Hugging Face hub
```

Comment:
Should we put a default/example value here too? Some people might not know where to look to find the correct CLIP Model ID